pkgs/by-name/qd/qdirstat/package.nix

```diff
@@ -51,7 +54,7 @@ mkDerivation {
     description = "Graphical disk usage analyzer";
     homepage = src.meta.homepage;
```

Comment:
```suggestion
    homepage = "https://github.com/shundhammer/qdirstat";
```

reviewers can easily click on this

Reply:
Should I make a treewide PR to replace `src.meta.homepage` and `inherit (src.meta) homepage` with URLs?

Reply:
We can do that eventually, we just don't need to add more occurrences of it.